content/java/concepts/output/terms/print/print.md

```diff
@@ -0,0 +1,45 @@
+---
+Title: '.print()'
+Description: 'Prints its argument to the console.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Output'
+  - 'Strings'
+  - 'Print'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The `.print()` method prints its argument to the console. Unlike the similar method [`.println()`](https://www.codecademy.com/resources/docs/java/output/println), `.print()` does not follow its argument with a new line, and any subsequent characters sent to the console will begin wherever the prior `.print()` command left off.
+
+## Syntax
+
+```pseudo
+System.out.print(argument)
+```
+
+`argument` will be displayed on the console.
+
+## Example
+
+The following example prints some content to the console.
+
+```java
+import java.io.*;
```

Comment:
@SSwiniarski I'm wondering if this line is even really necessary for these entries?

<img width="663" alt="Screen Shot 2022-06-24 at 8 27 59 AM" src="https://user-images.githubusercontent.com/23144845/175535769-280853b3-e1f1-421f-a6f8-d8c26feff525.png">